vendor/github.com/docker/docker/api/types/stats.go

```diff
@@ -50,6 +50,9 @@ type CPUStats struct {
 	// Online CPUs. Linux only.
 	OnlineCPUs uint32 `json:"online_cpus,omitempty"`
 
+	// Usage of CPU in %. Linux only.
+	CPU float64 `json:"cpu"`
```

Comment:
We can't update vendor/ - these are basically shared library files that are automatically updated with the library in question. If we need to change this we need to bring it into our code.

Reply:
right. Is there something similar I can get inspiration from?

Reply:
@baude Where should we keep structs like this?

@marusak Basically we just want to copy the struct into our codebase, with a note as to where it came from. Not sure exactly where it should end up yet, though.

Reply:
So I was looking into that a bit and I think I found some solution. If it is the right or best solution I cannot tell. Pushed it here for discussion. @baude WDYT?